src/bitshuffle_core.c

```diff
@@ -1138,7 +1138,7 @@ int64_t bshuf_untrans_bit_elem(const void* in, void* out, const size_t size,
 int64_t bshuf_blocked_wrap_fun(bshufBlockFunDef fun, const void* in, void* out, \
         const size_t size, const size_t elem_size, size_t block_size) {
 
-    size_t ii;
+    long long ii;
```

Comment:
Why?

Reply:
MSVC openmp doesn't like the size_t type

Reply:
But it is used throughout the code and is a very standard type. Why only this one place?

Reply:
The openmp pragma (as iterator)

Reply:
Okay fair enough. Maybe state this in a comment above this line so it doesn't bewilder future developers.